api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -588,7 +620,64 @@ public Locale getLocale() {
 		return Context.getLocale();
 	}
 	
+	/**
+	 * Change the current locale.
+	 * 
+	 * @param locale The locale.
+	 */
 	public void setLocale(Locale locale) {
 		this.locale = locale;
 	}
+	
+	/**
+	 * @return the value of the Global Propriety Handle Timezones <
```

Comment:
`Propriety`?

Reply:
And what does this mean? `Handle Timezones <`?

Reply:
I change it to `the value of the Global Property GP_TIMEZONE_CONVERSIONS`, what do you think?